app/com/linkedin/drelephant/util/InfoExtractor.java

```diff
@@ -39,32 +47,23 @@
   private static final Logger logger = Logger.getLogger(InfoExtractor.class);
   private static final String SPARK_EXTRA_JAVA_OPTIONS = "spark.driver.extraJavaOptions";
 
-  /**
-   * All the supported Schedulers.
-   * JobScheduler(x, y) where,
-   * x, is the unique property in the job conf that identifies the scheduler. This property must exist
-   *    in every job scheduled by the scheduler.
-   * y, is an instance of the Scheduler.
-   */
-  static enum JobScheduler {
-    AZKABAN(AzkabanScheduler.AZKABAN_JOB_URL);
+  private static final String SCHEDULER_CONF = "SchedulerConf.xml";
 
-    String identifier;
+  private static List<SchedulerConfigurationData> _schedulerConfData;
 
-    JobScheduler(String identifier) {
-      this.identifier = identifier;
-    }
+  public InfoExtractor() {
+    loadSchedulers();
```

Comment:
Yes you are right. `loadSchedulers()` was already being called somewhere else which is why things worked. Removed the dead code.